dev/WebView2/TestUI/WebView2Common.cs

```diff
@@ -0,0 +1,135 @@
+        }
+
+        public void ClearStatus()
+        {
+            Status1.Text = string.Empty;
+            Status2.Text = string.Empty;
+            Status3.Text = string.Empty;
+        }
+
+#if DESKTOP_APP
+        // In Desktop (Packaged) context, CWV2 can access the files directly from the app's Assets folder (and copying like the UWP case fails).
+        public void CopyTestPagesLocally()
+        {
+            Status1.Text = "HTML files ready";
+        }
+
+        // [Desktop (Packaged)] Get a file:// URI for the test page from app's Assets folder
+        // Ex: file:///C:/Program%20Files/WindowsApps/MUXControlsTestApp.Desktop_1.0.0.0_x86__8wekyb3d8bbwe/Assets/SimplePage.html
+        public static Uri GetTestPageUri(string testPageName)
+        {
+            string appDirectory  = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
+            string fullUri = @"file:///" + appDirectory + @"/Assets/" + testPageName;
+
+            return new Uri(fullUri);
+        }
+
+#else
+
+        readonly string[] TestPageNames =
+        {
+            "SimplePage.html",
+            "SimplePageWithButton.html",
+            "SimplePageWithScrollableColoredBlocks.html",
+            "SimplePageForFocus.html",
+            "SimplePageWithText.html",
+            "SimpleInputPage.html",
+            "SimplePageWithManyButtons.html",
+        };
+
+        // In UWP context, the CoreWebView2 can't load files from app-dir via file://, so copy them to %LOCALAPPDATA%
+        async public void CopyTestPagesLocally()
+        {
+            foreach (string fileName in TestPageNames)
+            {
+                // Read file from Appx
+                StorageFile sourceFile = await StorageFile.GetFileFromApplicationUriAsync(
+                                                new Uri("ms-appx:///Assets/" + fileName));
+                String sourceText = await FileIO.ReadTextAsync(sourceFile);
+
+                // Write file to Appdata - LocalFolder. Anaheim can read it via fully qualified file:// URI
+                StorageFolder storageFolder = Windows.Storage.ApplicationData.Current.LocalFolder;
+                StorageFile destFile = await storageFolder.CreateFileAsync(fileName, CreationCollisionOption.ReplaceExisting);
+                await FileIO.WriteTextAsync(destFile, sourceText);
+            }
+            Status1.Text = "HTML files ready"; // indicating all test files have been copied
+        }
+
+        // UWP Version: Get a file:// URI for the test page in app data (can be loaded by Anahaeim)
+        // Ex: file:///C:/Users/<user>/AppData/Local/Packages/MUXControlsTestApp_8wekyb3d8bbwe/TempState/SimplePage.htm
+        public static Uri GetTestPageUri(string testPageName)
+        {
+            string localAppDataPath = AppDataPaths.GetDefault().LocalAppData;
+            string fullUri = @"file:///" + localAppDataPath + @"/" + testPageName;
```

Comment:
nit you can make this a little more readable with an interpolated string
```suggestion
            string fullUri = $"file:///{localAppDataPath}/{testPageName}";
```